regcore/migrations/0002_auto_20160308_2228.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import mptt.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('regcore', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='regulation',
+            name='children',
+        ),
```

Comment:
There are a few more wrinkles here. Since we're replacing the `children` column with a backref in `models.py`, we don't have access to it in the migration--or, more precisely, `Regulation.children` is a `RelatedManager` in the migration, even before we add or drop columns. We can do this with raw sql queries, since the old `children` column still exists in the database--does a less ugly solution come to mind @cmc333333?